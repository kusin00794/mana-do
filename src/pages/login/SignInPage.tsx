import React from 'react';

import LoginForm from '../../modules/login/LoginForm';

const SignInPage = () => {
  return <LoginForm />;
};

export default SignInPage;
