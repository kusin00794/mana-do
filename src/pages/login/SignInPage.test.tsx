import { testSnapshots } from '../../utils/test';
import SignInPage from './SignInPage';

describe('<SignInPage />', () => {
  testSnapshots(SignInPage, [
    {
      props: {},
      description: 'default render',
    },
  ]);
});
