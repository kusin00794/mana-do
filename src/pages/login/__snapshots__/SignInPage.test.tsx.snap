// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInPage /> snapshots default render 1`] = `
<div
  className="login-form"
>
  <form
    className="row"
    onSubmit={[Function]}
  >
    <div
      className="col-12"
    >
      <label
        htmlFor="user_id"
      >
        Username (*) 
        <input
          id="user_id"
          name="userId"
          onChange={[Function]}
          value=""
        />
      </label>
    </div>
    <div
      className="col-12"
    >
      <label
        htmlFor="password"
      >
        Password (*)
         
        <input
          id="password"
          name="password"
          onChange={[Function]}
          type="password"
          value=""
        />
      </label>
    </div>
    <div
      className="col-12"
    >
      <button
        type="submit"
      >
        Sign in
      </button>
    </div>
  </form>
  <div
    className="toast info"
  >
    
  </div>
</div>
`;
